dv_utils/connectors/s3.json

```diff
@@ -0,0 +1,35 @@
+{
+  "schema": {
+    "description": {
+      "description": "Description of the files structure on S3.",
+      "required": true
+    },
+    "location": {
+      "description": "Location of the files on S3. */* patterns are supported. {model} can be used and will be replaced by the models listed in the data contract.",
+      "required": true
+    },
+    "file_format": {
+      "description": "Format of the files hosted on S3. csv, json or parquet are supported.",
+      "required": true
+    },
```

Comment:
does this mean we use one connector config to download multiple files that all have the same format? Not against it, just wondering if we cannot leverage the extension of the filename to be able to download multiple files in different formats with one connector (if this behaviour makes sense)

Reply:
usually it's all in the same format. I can explain